Gemfile

```diff
@@ -60,6 +60,8 @@ gem 'rexml'
 # Page profiler
 gem 'rack-mini-profiler'
 
+gem 'redis-rails'
```

Comment:
Do we actually still need this? [redis-rails' readme](https://github.com/redis-store/redis-rails) says that Rails 5.2.0 already includes built-in support for Redis cache.

Reply:
I am not 100% sure since it says we dont need the gemfile if we just need to store the fragment cache in Redis. I will merge this in first and we can check another time since we are also using redis for other purposes.